### PR TITLE
"pod plugins installed" subcommand

### PR DESCRIPTION
**Pull request by AliSoftware**
As per @orta 's suggestion in #37 
(Probably the first PR of 2015? :dancer: )

### CODE REVIEW

**Comment by orta** [comment]
can you tell if it has registered with the hooks manager?

**Comment by AliSoftware** [comment]
Probably. Will have to look at how to do that and how the HooksManager works first :wink:

**Comment by AliSoftware** [comment]
Plugins does not seem to register hooks unless they are actually loaded (by the Podfile for example).

So to ensure that a plugin is registered as a `:pre_install` or `:post_install` hook, one would need to load the plugin.
One may `require path/to/plugin/cocoapods_plugin.rb` just to ensure the plugin gets registered as a hook, but that seems dangerous to me as it will load/import the whole code from the plugin inside `Pod::Command::Plugins::Installed`… just to know if it registers with one or more hooks.

I'm afraid loading all plugins just for that may have side-effects, so I feel like it's a bad idea. What's your opinion?

**Comment by AliSoftware** [comment]
For reference, here is the code of the private method I wrote to list every hook a plugin is registered with:

```
# @return [Array<String>] List of hooks the given plugin did register for.
#
def registered_hooks(plugin_name)
  registrations = Pod::HooksManager.registrations
  return nil if registrations.nil?
  matching_hooks = registrations.values.filter { |hook| hook.plugin_name == plugin_name }
  matching_hooks.map(&:name)
end
```

The idea was to use this private method to list the registered hooks (if any) for each plugin listed by `pod plugins installed`. Unfortunately, those plugins will only be registered once loaded, so unless we somehow `require` them (in a sandbox env to avoid side-effects?), `Pod::HooksManager.registrations` seems to always return `nil`.

**Comment by orta** [comment]
Think it is reasonable to skip that for now. If we expand out the plugin API it's probably worth coming back to it.

**Comment by kylef** [comment]
Can we get some test coverage for this?

**Comment by AliSoftware** [comment]
Ok my bad plugins get registered (I was testing using `bundle exec …` and `cocoapods-keys` was not in my `Gemfile` that's the only reason why `Pod::HooksManager.registrations` were `nil`.

Will do another PR to add support for listing registered hooks.

**Comment by AliSoftware** [comment]
@kylef Agreed, definitely lacks specs.

**Comment by AliSoftware** [comment]
\ref #40

File: lib/pod/command/plugins/installed.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        def self.options
+          # Silent mode is meaningless for this command as
+          # the command only purpose is to print information
+          super.reject { |option, _| option == '--silent' }
+        end
+
+        def run
+          plugins = CLAide::Command::PluginManager.specifications
+          UI.title 'Installed CocoaPods Plugins:' do
+            if self.verbose?
```
Comment (kylef) [inline lib/pod/command/plugins/installed.rb]: Remember this is Ruby, self not required.

Comment (AliSoftware) [inline lib/pod/command/plugins/installed.rb]: o_O why did I write that? Too much holidays away from Ruby, I suppose :laughing:

**Pull request merged**